geonode/geonode/templates/subsites/holisoils/geonode-mapstore-client/snippets/custom_theme.html

```diff
@@ -1,6 +1,9 @@
 {% extends "geonode-mapstore-client/snippets/custom_theme.html" %}
 
 {% block content %}
+
+<link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;700&display=swap" rel="stylesheet">
```

Comment:
We should not fetch fonts from google. See https://www.maxgreger.de/wp-content/uploads/2023/03/Urteil-LG-Muenchen-I-Ismail-v.-30.03.2023-geschwaerzt.pdf

Reply:
If possible use a self-hosted version of it.

Reply:
Done, seems to work